.github/actions/composer-install/action.yml

```diff
@@ -0,0 +1,56 @@
+name: Composer Install
+
+inputs:
+  OP_SERVICE_ACCOUNT_TOKEN:
+    required: true
+    description: 1Password service account token to use for populating composer auth.json
+  OP_VAULT:
+    required: true
+    description: 1Password vault to use for populating composer auth.json
+  OP_ITEM:
+    required: true
+    description: 1Password item to use for populating composer auth.json
+
+runs:
+  using: "composite"
+  steps:
+    - name: Configure PHP environment
+      uses: shivammathur/setup-php@v2
+      with:
+        tools: composer:2
+        php-version: '8.2'
+
+    - name: Get Composer Cache Directory
+      id: composer-cache
+      shell: bash
+      run: |
+        echo "dir=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
+
+    - uses: actions/cache@v4
+      with:
+        path: ${{ steps.composer-cache.outputs.dir }}
+        key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-composer-
+
+    - name: Install 1Password CLI
+      uses: 1password/install-cli-action@v1
+
+    - name: Set env vars for 1Password CLI
+      shell: bash
+      run: |
+        echo "OP_VAULT=${{ inputs.OP_VAULT || 'Engineering' }}" >> $GITHUB_ENV
+        echo "OP_ITEM=${{ inputs.OP_ITEM || 'MT-Composer-Auth' }}" >> $GITHUB_ENV
+
+    - name: Create auth.json via 1Password CLI
+      shell: bash
+      env:
+        OP_SERVICE_ACCOUNT_TOKEN: ${{ inputs.OP_SERVICE_ACCOUNT_TOKEN }}
+      run: |
+        op user get --me
```

Comment:
Question: Should I leave this command before the `op inject...` in each instance? [It's output](https://github.com/moderntribe/moose/actions/runs/11943174941/job/33291712843#step:4:83) can be helpful for debugging to see who the authenticated 1Password user is, but it may also be more info than needed and "leak" sensitive info we don't need.